indexers/keyvalue/RedisDBIndexer/__init__.py

```diff
@@ -0,0 +1,51 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import json
+from typing import Optional
+
+
+from jina.executors.indexers.keyvalue import BinaryPbIndexer
+from google.protobuf.json_format import Parse
+from jina.proto import jina_pb2
+
+class RedisDBIndexer(BinaryPbIndexer):
+    """
+    :class:`RedisDBIndexer` Use Redis as a key-value indexer.
+    """
+
+    def get_add_handler(self):
+        """Get the database handler
+
+        """
+        try:
```

Comment:
I would remove the exception, just keep it as is, it would mostly be handled via docker image and we do not do these logging in other executors

Reply:
Ok, I'll remove it